airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -509,6 +506,12 @@ def create_job_yaml(self, context):
                         'annotations': {
                             'cluster-autoscaler.kubernetes.io/safe-to-evict':
                             'false'
+                        },
+                        'labels': {
+                            'app.kubernetes.io/name': self.dag_id,
+                            'app.kubernetes.io/part-of': self.part_of,
+                            'k8s.bluecore.com/team': self.team,
+                            'app.kubernetes.io/component': self.component,
```

Comment:
`k8s.bluecore.com/partner` should be set in here if `partner` is set on the operator.

Reply:
so it shouldn't be set otherwise?